Packs/PaloAltoNetworks_IoT3rdParty/Scripts/SendPANWIoTDevicesToCiscoISE/SendPANWIoTDevicesToCiscoISE.py

```diff
@@ -0,0 +1,92 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+GET_EP_ID_CMD = "cisco-ise-get-endpoint-id-by-name"
+ise_instance = demisto.args().get("active_ise_instance")
+attr_lists = demisto.args().get("device_maps")
+return_results("Processing %d devices" % len(attr_lists))
+count = 0
+
+'''
+Extract any connection error or error code if possible,
+Otherwise just return the original error
+'''
```

Comment:
please consider refactoring with a `main()` entry point, it is quite hard to track this code.

Reply:
added main()